packages/link/lib/link.module.scss

```diff
@@ -0,0 +1,5 @@
+@use '@dsco_/common/styles';
+
+.link {
+  @extend .dsco-link;
```

Comment:
this class is obviously useless right now, so it's a contrived example that i wanted to talk about:
- this results in less compiled CSS than if we were to import `@dsco_/common/styles` into `<Link>`
- we can apply any component-specific styling here. is that useful if we want `.dsco-link` to contain all relevant link styles?